pkg/flavor/kubernetes/multiwriter.go

```diff
@@ -5,36 +5,40 @@ package kubernetes
 import (
 	"fmt"
 	"github.com/container-storage-interface/spec/lib/go/csi"
+	//"github.com/docker/docker/pkg/mount"
 	log "github.com/hpe-storage/common-host-libs/logger"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 	apps_v1 "k8s.io/api/apps/v1"
 	core_v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/kubernetes/pkg/util/mount"
 	"os"
 	"strings"
+	"syscall"
 	"time"
 )
 
 const (
-	serviceNamePrefix    = "hpe-multiwriter-svc-"
-	deploymentNamePrefix = "hpe-multiwriter-"
-	pvcNamePrefix        = "hpe-multiwriter-pvc"
-	multiWriterNamespace = "kube-system"
-	nfsImage             = "quay.io/luis_pabon0/ganesha" // TODO evaluate latest nfs-ganesha image
-	deletionInterval     = 30                            // 60s with sleep interval of 2s
-	deletionDelay        = 2 * time.Second
-	creationInterval     = 60 // 120s with sleep interval of 2s
-	creationDelay        = 2 * time.Second
+	// TODO: fetch from cli/env args
+	multiNodePrefix    = "hpe-multinode-"
+	multiNodeNamespace = "hpe-multinode"
+	nfsImage           = "gcr.io/google_containers/volume-nfs:0.8"
+
+	deletionInterval = 30 // 60s with sleep interval of 2s
+	deletionDelay    = 2 * time.Second
+	creationInterval = 60 // 120s with sleep interval of 2s
+	creationDelay    = 2 * time.Second
 	// defaultVolumeSize is the implementation-specific default value in bytes
 	defaultVolumeSize = 10 * 1024 * 1024 * 1024 // 10 GiB
-	//defaultExportPath = "/exports"
+	defaultExportPath = "/"
+	readOnly          = "readOnly"
 )
 
-// CreateMultiWriterVolume creates multi-writer volume abstracting underlying nfs pvc, deployment and service
-func (flavor *Flavor) CreateMultiWriterVolume(request *csi.CreateVolumeRequest) (multiWriterVolume *csi.Volume, rollback bool, err error) {
-	log.Tracef(">>>>> CreateMultiWriterVolume with %s", request.Name)
-	defer log.Tracef("<<<<< CreateMultiWriterVolume")
+// CreateMultiNodeVolume creates multi-node volume abstracting underlying nfs pvc, deployment and service
+func (flavor *Flavor) CreateMultiNodeVolume(request *csi.CreateVolumeRequest) (multiNodeVolume *csi.Volume, rollback bool, err error) {
```

Comment:
Why did we change the function name to CreateMultiNodeVolume?  MultiAccess makes more sense to me if that's what your intent was.

Reply:
sure, was trying to align with flags and internal constants like MULTI_NODE_MULTI_WRITER, MULTI_NODE_READER_ONLY etc..

Reply:
oh i see.  then leave it as is